packages/dso-toolkit/src/namespaced-stylesheets.scss

```diff
@@ -1,7 +0,0 @@
-@import "@dso-toolkit/sources/src/components/conclusion/conclusion";
```

Comment:
Nog nadenken wat dit betekent en of we hier iets anders voor moeten maken

Reply:
@WiebeCnossen: In de vorige release heb ik deze file speciaal voor jou gerealiseerd, maar de verantwoordelijkheid voor het importeren van alle stylesheets is voor `dso-toolkit` 'n package opgeschoven:

van `dso-toolkit/src/namespaced-stylesheets.scss` naar `@dso-toolkit/sources/src/components`.

De semantiek van deze file is ook iets gewijzigd: Waar `namespaced-stylesheets.scss` alle stylesheets zou krijgen die in een later stadium voor `@use` gebruikt zouden worden, is `components.scss` alleen een lijstje van namespaced component mixins.